build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
 group = "net.leanix"
-version = "v0.6.9"
+version = "v1.0.0"
```

Comment:
Sorry if I missed this, but what changes qualified the move to version 1.0.0?

Reply:
Because we introduced a breaking change.

Reply:
Are you referring to this change? https://github.com/leanix/vsm-sbom-booster/commit/18ec68b08e6be9f47663788c16de03e2abfc0019